Makefile

```diff
@@ -98,6 +99,10 @@ BISON ?= bison
 STRIP ?= strip
 AWK ?= awk
 
+ifeq ($(ENABLE_FPIC),1)
```

Comment:
Instead of adding new option I would suggest adding :
```
ifeq ($(findstring CYGWIN,$(OS)),)
CXXFLAGS += -fPIC
endif
```
You can also make sure it only do not add it in case of cygwin and clang, since not sure if fPIC is at all affecting cygwin gcc build or is being ignored